speech/google/cloud/speech/client.py

```diff
@@ -46,26 +46,17 @@ class Client(client_module.Client):
 
     _connection_class = Connection
 
-    def async_recognize(self, content, source_uri, encoding, sample_rate,
+    def async_recognize(self, encoding, sample_rate, content=None,
```

Comment:
@tseaver haha O.o...

Since there are a few formats that are supported by the API, I'm not sure how we could actually detect the sample rate without pulling in a bunch of 3rd party libraries. 44100 is really common, so we could default to that if it's not passed in.